frontend/src/component/feature/EditFeature/EditFeature.tsx

```diff
@@ -54,7 +54,7 @@ const EditFeature = () => {
             await patchFeatureFlag(project, featureId, patch);
             navigate(`/projects/${project}/features/${name}`);
             setToastData({
-                title: 'Flag updated successfully',
+                text: 'Flag updated successfully',
```

Comment:
do we need to add successfully? in other command we normally just say what happened